ansible/roles/kafka-logging-setup/tasks/main.yml

```diff
@@ -1,15 +1,42 @@
 ---
-- name: Deploy Kafka cluster
+- name: Label Kafka nodes
+  k8s:
+    state: present
+      definition: "{{ lookup('template', 'node.yml.j2') }}"
+  with_items: "{{ kafka_nodes | default ([]) }}"
+
+- name: Deploy Kafka cluster with ephemeral storage
   k8s:
     state: present
     namespace: "amq"
     definition: "{{ lookup('file', item ) }}"
   with_items:
-    - kafka-cluster.yaml
-
-- name: Wait 30 seconds
+    - kafka-cluster-ephemeral.yaml
+  when: kafka_storage == 'ephemeral'
+
+- name: Deploy a kafka cluster with persistent storage
+  block:
+  - name: Create the local volumes for the brokers and zookeepers
+    k8s:
+      state: present
+      namespace: "amq"
+      definition: "{{ lookup('file', item ) }}"
+    with_items:
+      - kafka-lv-brokers.yaml
+      - kafka-lv-zookeepers.yaml
```

Comment:
Since the local-storage playbook will create these items we should remove this task and these templates right?

Reply:
done